packages/flutter_tools/lib/src/commands/ios.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+
+import '../runner/flutter_command_runner.dart';
+import '../runner/flutter_command.dart';
+
+class IOSCommand extends FlutterCommand {
+  final String name = "ios";
+
+  final String description = "Commands for creating and updating Flutter iOS projects";
+
+  final bool requiresProjectRoot = true;
+
+  IOSCommand() {
+    argParser.addFlag('init', help: 'Initialize the Xcode project for building the iOS application');
+  }
+
+  static Uri _xcodeProjectUri(String revision) {
+    return Uri.parse("https://storage.googleapis.com/flutter_infra/flutter/${revision}/ios/FlutterXcode.zip");
+  }
+
+  Future<List<int>> _fetchXcodeArchive() async {
+    print("Fetching the Xcode project archive from the cloud...");
+
+    HttpClient client = new HttpClient();
+    // TODO(chinmaygarde): Currently, engine releases are not tied to the release of the Xcode project
+    // archive. So use a "Current" tag. This will need to be replaced once released
+    // are in lockstep with the engine
+    HttpClientRequest request = await client.getUrl(_xcodeProjectUri("Current"));
+    HttpClientResponse response = await request.close();
+
+    if (response.statusCode != 200) {
+      throw new Exception(response.reasonPhrase);
+    }
+
+    BytesBuilder bytesBuilder = new BytesBuilder(copy: false);
+    await for (List<int> chunk in response) {
+      bytesBuilder.add(chunk);
+    }
+
+    return bytesBuilder.takeBytes();
+  }
+
+  Future<bool> _inflateXcodeArchive(String directory, List<int> archiveBytes) async {
+    print("Unzipping Xcode project to local directory...");
```

Comment:
Yeah, I was confused by what to use. init.dart was using regular print statements. But it was not clear which logging variants to use. I wish there was a simple method to log to either stdout or stderr